fastspring/lib/eventHandlers/handleOrderCreated.ts

```diff
@@ -4,6 +4,8 @@ import { createLicense } from "@shared-utils/licenseActions";
 import { insertUser } from "@shared-utils/userActions";
 import { getCustomAttributes, ORDER_ID_METADATA_KEY, SUBSCRIPTION_ID_METADATA_KEY } from "../utils/getCustomAttributes";
 
+import { components } from "@cryptlex/web-api-types/production";
+type LicenseBody = components["schemas"]["LicenseRequestModel"];
```

Comment:
@mumin-khan it is a type so better retain the name, LicenseBody -> LicenseRequestModel